### DISCUSSION TITLE
v2.0.0-rc.1

### DISCUSSION THREAD

**Discussion by peti2001** [codersrank-org/repo_info_extractor#189] [General]
Breaking changes:
- `./repo_info_extractor --repo_path ./path_to_repo` won't work anymore. Use this: `./repo_info_extractor local --repo_path ./path_to_repo`

Features:
- Add BitBucket support. Example usage: `./repo_info_extractor bitbucket --username="username" --password=password --visibility=private --emails=karakas.peter@gmail.com --skip_libraries --skip_update`
- Introduce commands concept. There are global and command level arguments
- Add `version` command
- Add `--skip_update` global argument
- Add `--skip_libraries` global argument 
- Add Blazor language support

<hr /><em>This discussion was created from the release <a href='https://github.com/codersrank-org/repo_info_extractor/releases/tag/v2.0.0-rc.1'>v2.0.0-rc.1</a>.</em>

**Comment by rahulrajsbkk** [comment]
When I am using it with bit bucket it only getting some repo which i don't have commits from my workplace its not getting the project wich i have commits

**Reply by peti2001** [reply]
Hi @rahulrajsbkk,
We use this API request: `https://api.bitbucket.org/2.0/repositories?role=contributor`, with the correct username and password (https://bitbucket.org/account/settings/app-passwords/) set as basic auth. 
According to the documentation, this will return with the repos you have write access to.

What is the difference between the repos that are extracted and those not considered?
